### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by CT15**
Work in progress

### CODE REVIEW

File: app/assets/stylesheets/scaffolds.scss [hunk 1]
```diff
@@ -0,0 +1,84 @@
+body {
```
Comment (donjar) [inline app/assets/stylesheets/scaffolds.scss]: File ini dibuang aja

File: db/migrate/20180131122521_create_feedbacks.rb [hunk 2]
```diff
@@ -1,7 +1,9 @@
 class CreateFeedbacks < ActiveRecord::Migration[5.1]
   def change
     create_table :feedbacks do |t|
+      t.string :name
       t.string :email
+      t.boolean :anonymous, :default => true
```
Comment (donjar) [inline db/migrate/20180131122521_create_feedbacks.rb]: `default: true`

File: app/assets/stylesheets/feedbacks.scss [hunk 3]
```diff
@@ -0,0 +1,47 @@
+// Place all the styles related to the Feedbacks controller here.
```
Comment (donjar) [inline app/assets/stylesheets/feedbacks.scss]: Spacing: pakai 2 spaces yah

File: app/views/feedbacks/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,56 @@
+<h2><p id="notice"><%= notice %></p></h2>
```
Comment (donjar) [inline app/views/feedbacks/show.html.erb]: Perhatiin spacing yang ini juga

**Comment by donjar** [comment]
Oh, and you guys also need to fix your Rubocop / Travis CI

**Comment by donjar** [comment]
Some more general comments:

- Instead of displaying anon? column di index, mendingan namanya tampilin yang asli, tapi kalo dia mau anon diganti dengan "Anonymous" gitu
- Specsnya dibenerin, coba jalanin `rspec`. View specsnya dibuang misalnya
- Emailnya divalidate mesti ada "@"

**Comment by CT15** [comment]
1. `rubocop autocorrect` selalu 'Inspecting 0 files' jadi aku harus manually `rubocop` dan betulin satu per satu. Do you have any idea why?
2. Testing untuk cek kalau nama dan email sudah dirubah jadi anonymous itu hrs di file apa? I removed it dari model/feedback_rspec.rb coz it does not seem to work (and I kinda know why). Anyway, black-box testing di localhost:3000 works so not really a big deal for now. Still, I would like to know how to do it.

THANKS!

**Comment by donjar** [comment]
1. Coba rubocop -a? Bukan autocorrect haha.
2. Depends perubahannya di mana. Diubahnya di model, view, atau controller?

**Comment by donjar** [comment]
And jangan lupa merge back from master

**Comment by CT15** [comment]
Errors in the controller Rspec I believe. Gara gara aku remove feedback root dari routes file.

**Comment by donjar** [comment]
Add `sign_in create(:user)` before your specs :)

**Pull request merged**